docs/modules/guards/key_concepts.md

```diff
@@ -0,0 +1,25 @@
+# Key Concepts
```

Comment:
Perhaps we could start a 'Moderation' or 'Alignment' module and put guard under it, since I can see other concepts being added.

Reply:
Yes I agree- I was just seeing guard as being that module. I made it it's own in the docs. I was thinking about calling it security but figured that blocking certain outputs can be used for more than  security- it can also be used to narrow what type of output you get to get a better output or certain type of output.

Maybe we should add a "security" section to the reference section and list all the ways, including guards, to avoid vulnerabilities? Also maybe talk about Eval best practices and storing keys in .env and such for newcomers as well as more advanced stuff as it comes up?

Reply:
I agree with the name alignment being better! I was originally thinking about calling it "security" but realized that guards can do stuff that's not security related like verifying that an output is good (not just safe) 

Should I rename and move the stuff from utils to an alignment section? 

Also, do you think it makes sense to change stuff from @RestrictionGuard to @AlignmentRestrict and so forth? I kinda think we should rename the module alignment to invite people to add more related tools but leave guards named the same because it's more descriptive and alignment is kinda a loaded term already.

I think we should have a separate section in "reference" called security. There we can talk about general stuff like .env and eval() safety for newcomers along with how to use guards/alignment tools specifically for safety.

Reply:
@pruksmhc I created a new module called "alignment" and added guards as a sub topic. I'm renaming this PR to match. LMK what you think of the alignment doc page. Would also love to hear any ideas you have. Even if we don't have time to add them this PR it would be cool to outline the work that still needs to be done to flesh out this module